action.yml

```
@@ -97,8 +96,12 @@ runs:
           ${{ inputs.extra-setup-args }}
       shell: bash
 
-    - name: Export KUBECONFIG environment variable
-      run: echo "KUBECONFIG=/etc/rancher/k3s/k3s.yaml" >> $GITHUB_ENV
+    - name: Prepare a kubeconfig in ~/.kube/config
+      run: |
+        mkdir -p ~/.kube
+        sudo cp /etc/rancher/k3s/k3s.yaml "$HOME/.kube/config"
+        sudo chown $(id -u):$(id -g) "$HOME/.kube/config"
```

```suggestion
        sudo cat /etc/rancher/k3s/k3s.yaml > "$HOME/.kube/config"
```
I think this can be simplified like this, but check it works!

---

Ohhh smart trick! I see why this may work, I think it will!